onnxruntime/core/providers/webnn/builders/impl/argmax_min_op_builder.cc

```diff
@@ -50,6 +50,11 @@ Status ArgMaxMinOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
   options.set("axes", axes);
   options.set("keepDimensions", keep_dims == 1);
   options.set("selectLastIndex", select_last_index == 1);
+  // TODO: use WebNN's opSupportLimits API to check the backend's supported output data types.
+  // If the backend doesn't support int64 output, we should use default int32 output data type
+  // then do a type casting (int32 -> int64) for the output. Refer to the CoreML EP for how to
```

Comment:
I suppose we should use the int32 output only if the dimension size being reduced is within that range. How about making this TODO comment simpler, i.e. just saying "check whether int64 output data type is supported by `opSupportLimits()`? Open an issue tracking this TODO and discuss the design details there.

Reply:
Make sense, I've updated the comment and created a new issue to track the TODO: https://github.com/microsoft/onnxruntime/issues/21401